doc/user/content/sql/create-sink/kafka.md

```diff
@@ -59,6 +59,7 @@ _item&lowbar;name_ | The name of the source, table or materialized view you want
 Field                | Value  | Description
 ---------------------|--------|------------
 `TOPIC`              | `text` | The prefix used to generate the Kafka topic name to create and write to.
+`COMPRESSION TYPE`   | `text` | The type of compression to apply to messages before they are sent to Kafka: `none`, `gzip`, `snappy`, `lz4`, or `zstd`. Default: `none`.
```

Comment:
```suggestion
`COMPRESSION TYPE`   | `text` | Default: `none`. The type of compression to apply to messages before they are sent to Kafka: `none`, `gzip`, `snappy`, `lz4`, or `zstd`.
```

Reply:
Fixed, thanks. (Well, at least, I realize I was doing it inconsistently with the rest. But putting the default before the explanation confuses my brain.)

Reply:
One of those things that is all over the place. I agree that it's weird to have it in the beginning; I'll straighten these out in _all_ pages this week.